website/docs/r/storage_account.html.markdown

```diff
@@ -136,14 +136,16 @@ The following arguments are supported:
 
 * `queue_properties` - (Optional) A `queue_properties` block as defined below.
 
-~> **NOTE:** `queue_properties` cannot be set when the `account_kind` is set to `BlobStorage`
+~> **NOTE:** `queue_properties` can only be set for `Standard` (`account_tier`) `StorageV2` (`account_kind`) Storage Accounts.
```

Comment:
Perhaps this'd be clearer as:

```suggestion
~> **NOTE:** `queue_properties` can only be configured when `account_tier` is set to `Standard` and `account_kind` is set to `StorageV2` Storage Accounts.
```

(assuming I've inferred that correctly from the message)?